actors/common/src/main/java/org/sunbird/feed/FeedUtil.java

```diff
@@ -42,9 +47,17 @@ public static Response saveFeed(ShadowUser shadowUser, String userId) {
     } else {
       Map<String, Object> data = feedList.get(index).getData();
       List<String> channelList = (List<String>) data.get(JsonKey.PROSPECT_CHANNELS);
-      if(!channelList.contains(shadowUser.getChannel())) {
+      if (!channelList.contains(shadowUser.getChannel())) {
         channelList.add(shadowUser.getChannel());
       }
+      List<Map<String, String>> orgList =
+          (List<Map<String, String>>) data.get(JsonKey.PROSPECT_CHANNELS);
+      orgList.forEach(
+          org -> {
+            if (!org.get("orgExtId").equals(shadowUser.getOrgExtId())) {
```

Comment:
orgExtId is no more. Please check by channel always and that's the only key.

Reply:
yes used channel instead orgExtId